Docker.DotNet/Models/ExecConfig.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace Docker.DotNet.Models
+{
+    public class ExecConfig
+    {
+        [DataMember(Name = "AttachStdin")]
+        public bool AttachStdin { get; set; }
+
+        [DataMember(Name = "AttachStdout")]
+        public bool AttachStdout { get; set; }
+
+        [DataMember(Name = "AttachStderr")]
+        public bool AttachStderr { get; set; }
+
+        [DataMember(Name = "Tty")]
+        public bool Tty { get; set; }
```

Comment:
for instance here if I don't send any value to this constructor, the value will be false.... However maybe in the actual REST call if I omit this field, it could be defaulting the `true` but here you're making the default `false`... :) In order to avoid that, please use `bool?` instead of `bool` and let users transmit this request without specifying any value for these bool fields.

Models directory is full of examples of this: https://github.com/ahmetalpbalkan/Docker.DotNet/blob/c2ca24da15ba1c12be8552ef52a58f07a111351f/Docker.DotNet/Models/RemoveContainerParameters.cs